cmd/oras/internal/option/common.go

```diff
@@ -28,23 +28,32 @@ const NoTTYFlag = "no-tty"
 
 // Common option struct.
 type Common struct {
-	Debug   bool
+	Debug bool
+	noTTY bool
+	// Verbose is deprecated. Use SuppressUntitled instead.
+	// SuppressUntitled=false is equivalent to Verbose=true, while Verbose=false
+	// no longer takes effect.
 	Verbose bool
-	TTY     *os.File
+
+	// SuppressUntitled suppresses the status output for untitled blobs.
+	// Default value: false.
+	SuppressUntitled bool
```

Comment:
What's the case of suppressing untitled status?

Reply:
But maybe we should refactor this part, that is to disable any status output when `--descriptor` is specified.

Reply:
> But maybe we should refactor this part, that is to suppress any status output when `--descriptor` is specified, no matter the content is titled or not.

It sounds like this refactoring can be done in the status output handlers. #1542

Reply:
> It sounds like this refactoring can be done in the status output handlers. https://github.com/oras-project/oras/issues/1542

The changes are overlapped. Status output handler is built upon `print.go`. #1542 aims to remove direct usage of command code on `print.go`. This PR aims to modify `print.go` directly.

The change proposed here is trying to reuse `untitled` state to avoid printing status for descriptor output. Suggest adding a temporary function in `print.go` to satisfy usage of `manifest push` and `blob push`, which will be removed after #1542 is done. In this way we can confidently remove all title-related code.